node/full_node_test.go

```diff
@@ -27,13 +27,15 @@ import (
 // simply check that node is starting and stopping without panicking
 func TestStartup(t *testing.T) {
 	ctx := context.Background()
-	_ = initAndStartNodeWithCleanup(ctx, t, "full").(*FullNode)
+	node := initAndStartNodeWithCleanup(ctx, t, "full").(*FullNode)
```

Comment:
The point was to use **test** assertions,  but this _type assert_ would panic in case of error (so next line is never reached). 
```suggestion
	node := initAndStartNodeWithCleanup(ctx, t, "full")
```